src/module.cc

```diff
@@ -49,6 +49,13 @@ namespace azplugins
     {
 namespace detail
     {
+
+void export_PotentialPairPerturbedLennardJones(pybind11::module&);
```

Comment:
Add forward declaration of the export method for your potential alphabetically to both of these lists. The first is the CPU version, the second is the GPU version. The code that implements this will be automatically generated by CMake.